Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:alpine
+
+RUN apk add python build-base linux-headers && npm install -g lerna@^3.22.1
+
+WORKDIR /usr/src/app
+
+COPY package.json package-lock.json lerna.json ./
+COPY common/package.json common/package-lock.json common/
+COPY frontend/package.json frontend/package-lock.json frontend/
+COPY server/package.json server/package-lock.json server/
+
+RUN lerna bootstrap --ci && lerna link
+
+COPY . .
+
+RUN lerna run build
+
+WORKDIR server
+CMD ["npm", "run", "run"]
```

Comment:
Should be changed to `npm run dev`

Reply:
I don't think it should

- This is running within the `server` package where there is not a `dev` script
- If we ran `npm run dev` from the root package then it would start a development server, which isn't what we want; we built the project in previous steps so that we can run the prod script here

Reply:
Ah you seem to be correct